### ISSUE TITLE
Clickable interactives lack accessibility features

### ISSUE THREAD

**Issue by ascholerChemeketa** [RunestoneInteractive/rs#1239]
Accessibility issues with clickable area interactives:

- [ ] There is no way to complete clickables using a keyboard.
- [ ] There is no screen reader feedback for interactions
